dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/UserController.java

```diff
@@ -281,96 +281,57 @@ private UserQueryParams makeUserQueryParams( WebOptions options )
     public void postXmlObject( HttpServletRequest request, HttpServletResponse response )
         throws Exception
     {
-        User user = renderService.fromXml( request.getInputStream(), getEntityClass() );
-
-        User currentUser = currentUserService.getCurrentUser();
-
-        if ( !validateCreateUser( user, currentUser ) )
-        {
-            return;
-        }
```

Comment:
FYI: the validate method would always throw an exception for invalid arguments. The returned boolean will never actually have any effect because it is either `true` or an exception is thrown. Therefore I removed the return value.